modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -69,6 +72,34 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
 
   const getTableFromCell = (cell: SugarElement<HTMLTableCellElement>) => TableLookup.table(cell, isRoot);
 
+  const performActionOnSelection = (action: ExecuteAction) => {
+    getSelectionStartCell(editor).each((cell) => {
```

Comment:
I guess some of the `{` and `}` could be removed here, since these are all one line in each indented function?

Reply:
I was going to, looked away, and then thought I'd done it. My bad.